src/pages/testcss/index.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react'
```

Comment:
we dont want this to be included in our sitemap or get crawled, please remove this page, you can go to implementation-guide.md to put the example of usage 🙏

Reply:
Yes, sure. I will remove this page. It was just a sample to show you.